x-pack/plugins/fleet/public/components/header.tsx

```diff
@@ -14,6 +14,12 @@ import type { EuiFlexItemProps } from '@elastic/eui/src/components/flex/flex_ite
 const Container = styled.div`
   border-bottom: ${(props) => props.theme.eui.euiBorderThin};
   background-color: ${(props) => props.theme.eui.euiPageBackgroundColor};
+
+  @media (max-width: 767px) {
+    .euiFlexItem {
+      margin-bottom: 0 !important;
```

Comment:
moved this to styled component, important is needed, because the original margin in eui has it too.